packages/celer-code-generator/src/koroks.csv

```diff
@@ -217,179 +217,179 @@ E10,1885.78,513.93,-2749.10,Balloon
+E15,1593.87,543.91,-2475.24,Confetti,Inside Goron's mouth
 E16,1865.55,533.93,-2465.04,Confetti,Top of skeleton
-E17,1867.81,511.00,-2400.63,Race,Glide over
-E18,1647.23,521.04,-2244.46,RockCircle,GG over
+E17,1867.81,511.00,-2400.63,Race
+E18,1647.23,521.04,-2244.46,RockCircle
 E19,1537.85,505.38,-2091.50,Balloon,High up
-E20,2019.41,557.72,-2172.89,BoulderGolf,Can go after bomb
+E20,2019.41,557.72,-2172.89,BoulderGolf
 E21,2170.73,547.03,-2109.55,BlockPuzzle
 E22,2355.87,524.65,-2067.37,LiftRock,On stone above ring
 E23,2663.89,564.49,-2066.87,LiftRockRubble
-E24,1412.02,408.74,-1759.80,BoulderGolf,WB away after
-E25,1544.33,394.20,-1837.21,Race,Glide and run
-E26,1687.55,427.39,-1717.08,BoulderGolf,Bomb down
+E24,1412.02,408.74,-1759.80,BoulderGolf
+E25,1544.33,394.20,-1837.21,Race
+E26,1687.55,427.39,-1717.08,BoulderGolf
 E27,1950.85,457.49,-1832.65,LiftRockRubble
-E28,2273.79,431.76,-1823.22,Balloon,Aim Above
+E28,2273.79,431.76,-1823.22,Balloon
 E29,2424.33,253.68,-1738.50,Basketball
 E30,2703.71,225.52,-1749.27,BlockPuzzle
-E31,1426.89,465.03,-1544.06,RockCircle,2 rocks
+E31,1426.89,465.03,-1544.06,RockCircle
 E32,2266.55,347.37,-1565.48,LiftRockRubble
-E33,2407.27,328.66,-1613.59,Race,.dir(N>) Turn
+E33,2407.27,328.66,-1613.59,Race
 E34,2751.06,293.76,-1675.90,RockCircle
 E35,1377.20,322.40,-1261.17,RockCircle
-E36,1940.14,226.01,-1217.73,LilyPads,No Drown
+E36,1940.14,226.01,-1217.73,LilyPads
 E37,2179.55,226.34,-1282.03,RockCircle
 E38,2478.57,313.41,-1391.59,BlockPuzzle
-E39,2752.29,233.72,-1366.33,Basketball,GG throw
+E39,2752.29,233.72,-1366.33,Basketball
 E40,2504.47,312.20,-1304.28,LiftRockRubble
-E41,2408.19,262.34,-1229.95,Balloon,Shoot from E42
-E42,2441.13,257.40,-1241.72,LiftRock,Ledge below stone thing
+E41,2408.19,262.34,-1229.95,Balloon
+E42,2441.13,257.40,-1241.72,LiftRock
 E43,2315.02,293.52,-1133.25,LiftRockRubble
-E44,2655.78,261.74,-971.17,LiftRock,Left set of trees
-E45,2275.00,134.98,-720.14,Race,ordinal DEFUSE
+E44,2655.78,261.74,-971.17,LiftRock
+E45,2275.00,134.98,-720.14,Race
 F01,1381.90,288.30,2277.44,AcornTree
-F02,1188.64,323.43,2476.70,Race,.dir(<S) Turn
+F02,1188.64,323.43,2476.70,Race
 F03,1305.28,191.34,2434.05,Balloon
-F04,1323.72,194.40,2453.45,RockCircle,Use tree
+F04,1323.72,194.40,2453.45,RockCircle
 F05,1462.67,336.79,2370.84,Balloon
-F06,2181.72,414.98,2534.69,LiftRock,Peak
+F06,2181.72,414.98,2534.69,LiftRock
 F07,2403.83,256.60,2511.22,LightChase
 F08,1966.62,365.96,2634.47,LiftRockTree
-F09,2099.72,363.66,2625.57,Basketball,2 Stasis hits
+F09,2099.72,363.66,2625.57,Basketball
 F10,1373.00,220.29,2749.04,FlowerChase,In cave
 F11,1936.52,492.98,2778.54,LiftRock,Peak
 F12,2103.91,422.50,2726.09,FlowerChase
 F13,2284.27,287.53,2659.97,LiftRock,Under tree
-F14,2874.91,299.73,2683.36,LilyPads,No Drown
+F14,2874.91,299.73,2683.36,LilyPads
 F15,1325.56,296.66,3008.67,LightChase
 F16,1562.56,298.62,2898.31,Race
-F17,1773.84,346.68,2930.86,Well,(Mag boulder in stone)
+F17,1773.84,346.68,2930.86,Well
```

Comment:
This the riola spring one. I wasn't sure if "Well" is the right classification so I put this here. I think this won't count as a "strat"

Reply:
Maybe something like "Ball and chain"?